src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.tsx

```
@@ -411,7 +408,7 @@ const ImageAnnotationModalMap = ({
         const pointStatus = properties?.isUnclassified ? 'unclassified' : confirmedStatus
         const popupContent = (
           <LabelPopup>
-            <IconCircle style={{ color: IMAGE_CLASSIFICATION_COLORS[pointStatus] }} /> {label}
+            <IconCircle style={{ color: COLORS[pointStatus] }} /> {label}
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)